### ISSUE TITLE
ObO Directory needs am Unspecified option

### ISSUE THREAD

**Issue by ColmDC** [SolidarityEconomyAssociation/owned-by-oxford-project#18]
**Describe the bug**
The primary filter in the ObO directory does not have an Unspecified option.

**To Reproduce**
Open the panel [here](https://maps.solidarityeconomy.coop/obo-public/?showDatasetsPanel=false&disableClusteringAtZoom=12&maxZoomOnOne=8)

**Expected behavior**
We can select all the initiaties that don't have a primary activity specified.

**Comment by ColmDC** [comment]
Is this still an issue for OBO, given that we no merge primary and seondary activities in the directory filter, so orgs like Orinoco Scrapstore appear as they have their seconary activities set. I guess there could be some that have neither primary nor secondary set.

**Comment by ColmDC** [comment]
Headington Fairtrade has neither set.

**Comment by ColmDC** [comment]
Have all the recent changes made this easier to address? @wu-lee

**Comment by wu-lee** [comment]
This seems like a new feature? I don't remember being able to do this.

The old Oxford SE map can't do it either: https://oxford.solidarityeconomy.coop/map.html

**Comment by ColmDC** [comment]
Agreed, it should not be classified as a bug. I'll move it back to general backlog. Q still stands though, do all the recent changes make this any easier to support?

**Comment by ColmDC** [comment]
It exists in the co-ops UK map, because _unspecified_ is an explicit option in the local vocab used in the primary data. Okay, also dropping this from the Epic too.

**Comment by wu-lee** [comment]
The recent changes don't especially make it more easy, I think, other than just clarifying a few dark corners for me.

So one way is to explicitly include a term in the vocab, as Co-ops UK does it. (And ensure all initiatives have that, rather than being blank; also it can't be  mixed with other terms.)

If we do it with some special logic, like "All Items", there is a problem of localisation, mentioned in https://github.com/digitalcommons/mykomap/issues/177 which would equally apply here.  That is, if there's a term we want to show which isn't in the vocab, like "All Countries" or "Unspecified", where do the language translations come from?

Currently we have hard-wired translations for a fixed number of phrases in a fixed number of languages. This works for now, but will get fragile/unwieldy if we support more languages and/or special cases.

An additional problem of "All Countries" is: there is a hack in the code to detect when to use it rather than the generic "All Items". So we'd need to find a solution which doesn't create more hacks like that.

If we fix the new problem of unlocalised field names in the search filters, this might provide a way to solve this one too.
